kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/message/store/provider/MessageStore.java

```diff
@@ -37,23 +37,28 @@ public interface MessageStore {
      * the <code>createdOn</code> message parameter to the current time.
      * <br>
      * 
-     * @param topic    the value of the <code>topic</code> parameter.
-     * @param payload  topic the value of the <code>payload</code> parameter.
-     * @param qos      topic the value of the <code>QoS</code> parameter.
-     * @param retain   topic the value of the <code>retain</code> parameter.
-     * @param priority topic the value of the <code>priority</code> parameter.
+     * @param topic
+     *            the value of the <code>topic</code> parameter.
```

Comment:
It looks like this file has only formatting related changes, if this is true can you please roll them back?